server/ips_test.go

```diff
@@ -0,0 +1,266 @@
+package server
+
+import (
+	"fmt"
+	"github.com/status-im/status-go/server/servertest"
+	"net"
+	"testing"
+
+	"github.com/stretchr/testify/suite"
+	"go.uber.org/zap"
```

Comment:
```suggestion
	"fmt"
	"net"
	"testing"

	"github.com/stretchr/testify/suite"
	"go.uber.org/zap"
	
	"github.com/status-im/status-go/server/servertest"
```

This order should satisfy the linter